src/sentry/similarity/__init__.py

```diff
@@ -57,21 +60,31 @@ def get_frame_attributes(frame):
     return attributes
 
 
-features = FeatureSet(
-    MinHashIndex(
-        redis.clusters.get(
-            getattr(
+def _make_index(cluster=None):
+    if not cluster:
+        try:
+            cluster_id = getattr(
                 settings,
                 'SENTRY_SIMILARITY_INDEX_REDIS_CLUSTER',
-                'default',
-            ),
-        ),
+                'similarity',
+            )
+            cluster = redis.redis_clusters.get(cluster_id)
+        except KeyError:
+            logger.info('No redis cluster provided for similarity, using NOOP index.')
```

Comment:
I'd instantiate the client here and just us the `repr` value of it instead of "NOOP index" here.

Reply:
f0d0b3cd161b711cc5dd25fda0c2c0feee28b80f